components/navigation.tsx

```
@@ -0,0 +1,274 @@
+]
+
+export function Navigation({ theme = 'dark', connectionStatus = 'connecting' }: NavigationProps) {
+  const pathname = usePathname()
+  const [isMobileMenuOpen, setIsMobileMenuOpen] = useState(false)
+
+  const handleQuickAction = (action: string) => {
+    // Handle quick actions
+    switch (action) {
+      case 'ai-trading':
+        // Scroll to AI trading panel or open modal
+        const aiPanel = document.querySelector('[data-component="ai-trading-panel"]')
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)